web/rootfs/defaults/settings-config.js

```diff
@@ -304,3 +304,8 @@ if (!config.testing.hasOwnProperty('octo')) config.testing.octo = {};
 
 config.testing.capScreenshareBitrate = {{ $TESTING_CAP_SCREENSHARE_BITRATE }};
 config.testing.octo.probability = {{ $TESTING_OCTO_PROBABILITY }};
+
+{{ if $DISABLE_DEEP_LINKING -}}
```

Comment:
Assign it unconditionally please, since we are using a default value on the template.